test/Device.test.ts

```diff
@@ -16,7 +16,11 @@ suite('IoT Device Workbench: Device', () => {
   test('property of device should be set correctly', function(done) {
     const context = new TestExtensionContext();
     const channel = vscode.window.createOutputChannel('IoT workbench test');
-    const device = new AZ3166Device(context, channel, '', []);
+    const telemetryContext: TelemetryContext = {
+      properties: {result: 'Succeeded', error: '', errorMessage: ''},
+      measurements: {duration: 0}

```

Comment:
will we set actual duration value in the code?

Reply:
This is an initialization status of telemetryContext. The same as we initialized in extension.ts.